zebra-state/src/constants.rs

```diff
@@ -13,3 +15,6 @@ pub const MAX_BLOCK_REORG_HEIGHT: u32 = MIN_TRANSPARENT_COINBASE_MATURITY - 1;
 
 /// The database format version, incremented each time the database format changes.
 pub const DATABASE_FORMAT_VERSION: u32 = 4;
+
+/// Portion of OS error when the RocksDB lock file is already open.
```

Comment:
This text is not an OS-specific error, it comes from RocksDB.
```suggestion
/// Part of the RocksDB error when its lock file is already open.
```